go/libraries/doltcore/sqle/dfunctions/merge.go

```diff
@@ -30,26 +31,35 @@ import (
 const MergeFuncName = "merge"
 
 type MergeFunc struct {
-	expression.UnaryExpression
+	children []sql.Expression
 }
 
 // NewMergeFunc creates a new MergeFunc expression.
-func NewMergeFunc(e sql.Expression) sql.Expression {
-	return &MergeFunc{expression.UnaryExpression{Child: e}}
+func NewMergeFunc(args ...sql.Expression) (sql.Expression, error) {
+	return &MergeFunc{children: args}, nil
 }
 
 // Eval implements the Expression interface.
 func (cf *MergeFunc) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
-	val, err := cf.Child.Eval(ctx, row)
-	if err != nil {
-		return nil, err
-	} else if val == nil {
-		return nil, nil
-	}
-
 	sess := sqle.DSessFromSess(ctx.Session)
-	if sess.Username == "" || sess.Email == "" {
-		return nil, errors.New("commit function failure: Username and/or email not configured")
+
+	// Get the params associated with MERGE. We can share the commit argparser for now.
```

Comment:
I think it's because the Merge in commands.go and the Merge in dfunctions.go support different arguments. This could trip up the HelpAndUsagePrinters function. 

I am going to leave this as a TODO for now, and do some refactoring in a future PR.